docs/application/native/tutorials/details/app-filtering.md

```diff
@@ -240,12 +240,6 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/opengles.version.3_0` | Specify this key, if the application requires OpenGL&reg; ES version 3.0.You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 2.4   |
```

Comment:
Line 82:  The following table lists the available screen size features.
Please add colon at the end of the sentence.
Line 145: If you want to check which features are necessary for using a specific API, see the related feature in the native API Reference.
Please change "check" word to "verify"


Line 145: The following tables show the available requirements for an application package. If you want to check which features are necessary for using a specific API, see the related feature in the native API Reference.

Please change as:
The following tables shows the available requirements for an application package:

Suggestion:
Please can you add this line "If you want to check which features are necessary for using a specific API, see the related feature in the native API Reference." after the "Table: Available mobile native requirements"

Please change word "check" to "verify"

Line 239: Specify this key, if the application requires OpenGL&reg; ES version 2.0.You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered.

Please change as:
Specify this key, if the application requires OpenGL&reg; ES version 2.0. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered.